orcid_hub/templates/profile_entry.html

```diff
@@ -8,7 +8,9 @@
         <h1>{{tile or ("Education" if section_type == "EDU" else "Funding" if section_type == "FUN" else "Peer Review" if
           section_type == "PRR" else "Work" if section_type == "WOR" else "Researcher Url" if section_type == "RUR" else
           "Other Name" if section_type == "ONR" else "Keyword" if section_type == "KWR" else
-          "Address" if section_type == "ADR" else "Other IDs" if section_type == "EXR" else "Employment")}}</h1>
+          "Address" if section_type == "ADR" else "Other IDs" if section_type == "EXR" else "Distinction" if
+          section_type == "DST" else "Membership" if section_type == "MEM" else "Service" if section_type == "SER" else
+          "Qualification" if section_type == "QUA" else "Invited Position" if section_type == "POS" else "Employment")}}</h1>
```

Comment:
improve the layout and add the same user data that are shown in the section forms ... refactor to make it as a reusable macro (the user basic data: name, orcid, email)

![form](https://user-images.githubusercontent.com/177266/60229110-e39c0b80-98e8-11e9-9896-4d3cba1fc766.png)

- external IDs on the right side;
- the other fields all can be placed on the left side ...
- "Cancel" can be red color;